appveyor.yml

```diff
@@ -1,7 +1,7 @@
 environment:
   matrix:
-  - julia_version: 0.7
```

Comment:
0.7 is supported, it should still be tested

Reply:
@ranjanan gave me a link where there was no test against it in appveyor that is why I removed it. I added 0.7 again.